go-controller/pkg/kubevirt/switch.go

```diff
@@ -0,0 +1,22 @@
+package kubevirt
+
+import (
+	"net"
+	"strings"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+const (
+	ARPProxyIPv4 = "169.254.1.1"
+	ARPProxyIPv6 = "d7b:6b4d:7b25:d22f::1"
+)
+
+// ComposeARPProxyLSPOption returns the "arp_proxy" field needed at router type
+// LSP to implement stable default gw for pod ip migration, it consiste of
```

Comment:
The logical flows generated for arp_proxy has less priority than the ones generated for normal switch ARPs, so they are a fallback in case the node do not "own" the subnet

https://github.com/ovn-org/ovn/commit/9e3429289589d2cb73d12f2b5d57ea9eb7ec638d